opengrok-indexer/src/main/java/org/opengrok/indexer/index/IndexDatabase.java

```diff
@@ -1588,6 +1611,7 @@ public static Definitions getDefinitions(File file)
             return null;
         }
 
+        Document doc;
```

Comment:
Maybe move the `return` as the last statement of the try, so this declaration and assignment can also be moved together?